gas/v1.25.0.json

```diff
@@ -0,0 +1,1825 @@
+{
+  "feature_version": 28,
```

Comment:
Shouldn't this be bumped to 29 for 1.25?
https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-gas-schedule/src/ver.rs#L91

Reply:
Force-pushed the fix